debian/changelog

```diff
@@ -1,6 +1,10 @@
 matrix-synapse-py3 (1.47.0+nmu1) UNRELEASED; urgency=medium
 
   * Update scripts to pass Shellcheck lints.
+  * Remove unused Vagrant scripts from debian/ directory.
+  * Change package Architecture to any.
```

Comment:
it might be good to expand this to explain the visible effect, rather than just what was changed

Reply:
yeah *I* know what the difference is :)